whois-commons/src/main/java/net/ripe/db/whois/common/etree/IntervalMap.java

```diff
@@ -161,4 +161,20 @@ public interface IntervalMap<K extends Interval<?>, V> {
      *         intervals.
      */
     List<V> findExactAndAllMoreSpecific(K key);
+
+    /**
+     * Finds all values associated with intervals that are contained within (more specific than)
+     * and whose prefix is most high
+     * <code>key</code>.
```

Comment:
Is it possible to use findFirstMoreSpecific ? It says " The resulting values are ordered from least specific interval to most specific interval"

Reply:
Discussed in previous comment

Reply:
noup
We need to hit the last leafs of the tree. `findFirstMoreSpecific` will give you just the first more specific
We need to get the last more specific lets say...we need to get the children without children (last leafs of the tree)

Reply:
I gave you an example in slack